riva/clients/asr/riva_streaming_asr_client.cc

```diff
@@ -137,6 +140,7 @@ main(int argc, char** argv)
   str_usage << "           --stop_history_eou=<int>" << std::endl;
   str_usage << "           --stop_threshold=<float>" << std::endl;
   str_usage << "           --stop_threshold_eou=<float>" << std::endl;
+  str_usage << "           --custom_configuration=<key:value,key:value>" << std::endl;
```

Comment:
we can say `<key:value,key:value,...>` just for better clarity that it can be any number of key/value pairs.

Reply:
Updated